web-client/src/presenter/actions/Login/forgotPasswordAction.tsx

```diff
@@ -0,0 +1,51 @@
+import { state } from '@web-client/presenter/app.cerebral';
+import React from 'react';
+
+export const forgotPasswordAction = async ({
+  applicationContext,
+  get,
+  path,
+}: ActionProps) => {
+  const { email } = get(state.authentication.form);
+
+  try {
+    await applicationContext
+      .getUseCases()
+      .forgotPasswordInteractor(applicationContext, { email });
+
+    return path.success({
+      alertSuccess: {
+        message: (
+          <>
+            If there is a DAWSON account for {email} we’ll send an email with a
+            code. This code will expire in 1 hour. If you don’t see the email,
```

Comment:
is this really 1 hour or 24 hours? Elsehwere in the codebase it seems to suggest 24...

Reply:
Hmm, I think see it's the verify email address link that's 24 hours, but the forgot password code is 1 hour.